src/scalarstats.jl

```diff
@@ -153,9 +182,21 @@ end
 #
 #############################
 
+"""
+    percentile(v, p)
+
+Return the `p`th percentile of a real-valued array `v`.
+"""
 percentile{T<:Real}(v::AbstractArray{T}, p) = quantile(v, p * 0.01)
 
 quantile{T<:Real}(v::AbstractArray{T}) = quantile(v, [.0, .25, .5, .75, 1.0])
+
+"""
+    nquantile(v, n)
```

Comment:
I mean that the default value is `n=4`, even if that's implemented using a separate method (the one above). So better show this in the signature.